Support for Federations and Trust Schemes, removal of termsOfUse (#43)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

}
```

> ## Support for Federations and Trust Schemes

Author: I'd change Trust Schemes to Trust Models if you agree

I heard some comments about the concrete meaning of the term trust scheme

Reviewer: why not trust framework? what is the distinction/definition of trust scheme vs trust model vs trust framework?

Author: I'm in favor of Trust Framework, I'm doing this change

Author: Done c72b02f0791d0de05d5428a39c3672b1e4f58636

> The Verifier who wants to obtain Verifiable Credentials issued by Credential Issuer who is a participant of a federation, or adheres to a known trust scheme, MUST create a `presentation_definition` that includes a known property within such federation, that allows to distinguish the request according to it.

Author: I definitively have to change this text

Author: Done

> To facilitate this, federations need to determine how a Credential Issuer indicates in a Verifiable Credential that it is a member of one or more federations. Once this is done, the Verifier will be able to create a `presentation_definition` that includes this filtering criteria. 

Reviewer: what if there is no explicit claim in the credential that conveys the federation that a credential belogs to? I am not sure the verifier can communicate using PE that "please return this credential if after obtaining the trust chain using the iss value in the credential, you are certain that the credential belongs to Federation A"?

Author: I'd not mention any trust chain or policy since these devices are not required here but Just a shared and agreed trust framework where many entities may find themselves under the same regulation

Reviewer: the text you have written mandates/assumes that issuers put inside a credential information about which federation that issuer belongs to, I just do not think that's the case. there are cases where `iss` in the credential is HTTPS and a file found by appending a `.well-known` is sufficient. in other places, `iss` is a DID and metadata with the federation information is hosted at the service endpoint obtained from a DID Document.

I suggest you file an issue first. and this PR is refactored to remove dependency on `termsofuse` property.

> To facilitate this, federations will need to determine how a Credential Issuer can indicate in a Verifiable Credential that they are a member of one or more federations. Once this is done, the Verifier will be able to create a `presentation_definition` that includes this filtering criteria. 

Author: ```suggestion
To facilitate this, federations need to determine how a Credential Issuer indicates in a Verifiable Credential that it is a member of one or more federations. Once this is done, the Verifier will be able to create a `presentation_definition` that includes this filtering criteria. 
```

> To facilitate this, federations will need to determine how a Credential Issuer can indicate in a Verifiable Credential that they are a member of one or more federations. Once this is done, the Verifier will be able to create a `presentation_definition` that includes this filtering criteria. 

Reviewer: I think an example of how to use a filter property would be helpful?
so the idea is that the credential contains a claim that expresses a trust framework that the issuer of that credential belongs to and verifier must know that claim and put restrictions on that claim using PE, correct? if so, explaining this a little more would help.

Author: I agree, I'll try to put in the form of comments some proposal before the commit for your kindly review
